c/MemSafety-Arrays.set

```diff
@@ -2,6 +2,9 @@
 array-memsafety/*.yml
 array-examples/*.yml
 array-memsafety-realloc/*.yml
+termination-crafted/Arrays*.yml
```

Comment:
This line matches also files like:
```
termination-crafted/Arrays01-EquivalentConstantIndices-1.yml
termination-crafted/Arrays01-EquivalentConstantIndices-2.yml
```
which have no memsafety property.

Reply:
Where exactly is the problem? If there is no memsafety property that means the benchmark is not used in the MemSafety category.

The line just means where are you looking for benchmarks. E.g. `array-examples/*.yml` means too `array-examples/data_structures_set_multi_proc_ground-1.yml`.

Reply:
Well, I would suppose that when a benchmark is in a .set file for a category, then it should contain a label with the verdict whether it is true or false for the property of the category.

Reply:
I did not find such a requirement in the rules, but every (sub)category in https://sv-comp.sosy-lab.org/2021/benchmarks.php has assigned a specification _for all_ the tasks matching the pattern in the set file.

For example:
```
MemSafety-TerminCrafted

The verification tasks consist of the programs that match

# Contains tasks for checking memory safety of array-based programs.
termination-crafted/*.yml

with the specification:

CHECK( init(main()), LTL(G valid-free) )
CHECK( init(main()), LTL(G valid-deref) )
CHECK( init(main()), LTL(G valid-memtrack) )
```
So if there is a file that does not have the property in its .yml file but matches the pattern, it contradicts the information on this page.

Reply:
Well, if you want to take it literally, you have to check all `*.set`, because it is usual to have similar programs of different versions in the same directory ( reachsafety violation, memsafety violation, ..., otherwise undefined behavior). I would leave it for another PR.

> # Contains tasks for checking memory safety of array-based programs.

This is not true either. There are programs where the array is not used.

Reply:
> Well, if you want to take it literally, you have to check all *.set

Agree, just why not start here and now?

Anyway, I do not want to block the PR just because of this. I'm ok with merging it as it is.

Reply:
So  I leave it for next PR.